### ISSUE TITLE
MAL search  bug

### ISSUE THREAD

**Issue by ax-lord** [mProjectsCode/obsidian-media-db-plugin#36]
- [x] The Plugin is up to date
- [x] Obsidian is up to date

**Describe the bug**
```
plugin:obsidian-media-db-plugin:3483 TypeError: Cannot read properties of null (reading 'toLowerCase')
    at MALAPI.eval (plugin:obsidian-media-db-plugin:2718:56)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:obsidian-media-db-plugin:50:24)
```

**To Reproduce**

**This error appears when I search for anime such as:**

Akira
Rakuen Tsuihou
Fate/kaleid liner Prisma☆Illya: Phantasm
Kimi no Suizou wo Tabetai
Kono Sekai no Katasumi ni 
Kono Subarashii Sekai ni Shukufuku wo! Kurenai Densetsu
Sayonara no Asa ni Yakusoku no Hana wo Kazarou
Riding Bean
Summer Ghost
Bakemono no Ko
Tobira wo Akete
Tenki no Ko
Mimi wo Sumaseba

**Screenshots**
![image](https://user-images.githubusercontent.com/106933377/176988972-f061f1f3-bb32-4584-a400-227490113509.png)

![image](https://user-images.githubusercontent.com/106933377/176988865-724fd007-9c5e-41cf-aeea-492bc81755be.png)

**Occurs on**
- [x] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Plugin version**
0.3.0

**Comment by mProjectsCode** [comment]
It seems like this happens because the API returns no type for the anime. 
I will try to get a release out today, that fixes this.

**Comment by mProjectsCode** [comment]
should be fixed in release `0.3.1`
